lib/customer-deposit-wallet/http/Cardano/Wallet/Deposit/HTTP/Types/OpenAPI.hs

```diff
@@ -217,12 +219,26 @@ chainPointAtSlotSchema =
     mempty
         & type_ ?~ OpenApiObject
         & properties
-            .~ [ ("slot_no", Inline slotSchema)
-               ]
+            .~  [ ("slot_no", Inline slotSchema)
+                , ("header_hash", Inline headerHashSchema)
+                ]
 
 slotSchema :: Schema
 slotSchema =
     mempty
         & type_ ?~ OpenApiInteger
         & minimum_ ?~ 0
         & maximum_ ?~ fromIntegral (maxBound :: Word64)
+
+headerHashSchema :: Schema
+headerHashSchema =
+    blake2b_256Schema
+        & description ?~ "Hash (Blake2b_256) of a block header."
+
+blake2b_256Schema :: Schema
+blake2b_256Schema =
+    mempty
+        & type_ ?~ OpenApiString
+        & format ?~ "hex"
+        & minLength ?~ (2*32)
```

Comment:
what about introducing `charsPerByte = 2` and internally `bytePayload=32` Then it would be very clear and well documented imho;-)

Reply:
I have added a definition `hexCharactersPerByte`.